Resources/doc/internals/reference_configuration.md

```diff
@@ -14,6 +14,9 @@ hwi_oauth:
             scope:               "user:email"
             options:
                 csrf:            true
+                state:
+                  some:          parameter
+                  some-other:     parameter
```

Comment:
You may align the value with the one above.

Reply:
@XWB Yupp, done, thank you.